cinema/management/commands/wait_for_db.py

```diff
@@ -0,0 +1,21 @@
+import time
+
+from django.db import connections
+from django.db.utils import OperationalError
+from django.core.management.base import BaseCommand
+
+
+class Command(BaseCommand):
+    help_ = "Waits for the database to be available"
+
+    def handle(self, *args, **kwargs):
+        self.stdout.write("Waiting for database...")
+        db_conn = None
+        while not db_conn:
+            try:
+                db_conn = connections["default"]
+                db_conn.cursor()
+            except OperationalError:
+                self.stdout.write("Database unavailable, waiting 1 second...")
+                time.sleep(1)

```

Comment:
Consider adding a maximum number of attempts or a timeout for the database connection. In the current implementation, the command will wait indefinitely if the database is not available. This could cause the application to hang if there's an issue with the database. Adding a limit for the number of attempts or a timeout can help to prevent this.